Wikipedia/Networking/Fetchers/WMFENFeaturedTitleFetcher.m

```diff
@@ -0,0 +1,175 @@
+
+@interface WMFENFeaturedTitleRequestSerializer : AFHTTPRequestSerializer
+@end
+
+@interface WMFENFeaturedTitleResponseSerializer : WMFApiJsonResponseSerializer
+@end
+
+@interface WMFTitlePreviewRequestSerializer : AFHTTPRequestSerializer
+@end
+
+@interface WMFENFeaturedTitleFetcher ()
+@property (nonatomic, strong) AFHTTPRequestOperationManager* featuredTitleOperationManager;
+@property (nonatomic, strong) AFHTTPRequestOperationManager* titlePreviewOperationManager;
+@end
+
+@implementation WMFENFeaturedTitleFetcher
+
++ (AFHTTPRequestOperationManager*)featuredTitleOperationManager {
+    AFHTTPRequestOperationManager* featuredTitleOperationManager = [AFHTTPRequestOperationManager wmf_createDefaultManager];
+    featuredTitleOperationManager.requestSerializer  = [WMFENFeaturedTitleRequestSerializer serializer];
+    featuredTitleOperationManager.responseSerializer = [WMFENFeaturedTitleResponseSerializer serializer];
+    return featuredTitleOperationManager;
+}
+
++ (AFHTTPRequestOperationManager*)titlePreviewOperationManager {
+    AFHTTPRequestOperationManager* titlePreviewOperationManager = [AFHTTPRequestOperationManager wmf_createDefaultManager];
+    titlePreviewOperationManager.requestSerializer  = [WMFTitlePreviewRequestSerializer serializer];
+    titlePreviewOperationManager.responseSerializer =
+        [WMFMantleJSONResponseSerializer serializerForValuesInDictionaryOfType:[MWKSearchResult class] fromKeypath:@"query.pages"];
+    return titlePreviewOperationManager;
+}
+
+- (instancetype)init {
+    self = [super init];
+    if (self) {
+        self.featuredTitleOperationManager = [WMFENFeaturedTitleFetcher featuredTitleOperationManager];
+        self.titlePreviewOperationManager  = [WMFENFeaturedTitleFetcher titlePreviewOperationManager];
+    }
+    return self;
+}
+
+- (AnyPromise*)featuredArticlePreviewForDate:(nullable NSDate*)date {
+    @weakify(self);
+    MWKSite* site = [MWKSite siteWithLanguage:@"en"];
+    return [self.featuredTitleOperationManager wmf_GETWithSite:site parameters:date operation:nil]
+           .thenInBackground(^(NSString* title) {
+        @strongify(self);
+        if (!self) {
+            return [AnyPromise promiseWithValue:[NSError cancelledError]];
+        }
+        return [self.titlePreviewOperationManager wmf_GETWithSite:site parameters:title operation:nil]
+        .then(^(NSArray<MWKSearchResult*>* featuredTitlePreviews) {
+            return featuredTitlePreviews.firstObject;
+        });
+    });
+}
+
+@end
+
+@implementation WMFENFeaturedTitleRequestSerializer
+
++ (NSDateFormatter*)featuredArticleDateFormatter {
+    static NSDateFormatter* feedItemDateFormatter;
+    static dispatch_once_t onceToken;
+    dispatch_once(&onceToken, ^{
+        feedItemDateFormatter = [[NSDateFormatter alloc] init];
+        feedItemDateFormatter.dateFormat = @"MMMM d, YYYY";
+        // feed format uses US dates
+        feedItemDateFormatter.locale = [NSLocale localeWithLocaleIdentifier:@"en-US"];
```

Comment:
@bgerstle I don't think `en-US` is a locale? This is probably evaluating to nil. `en_US` is a locale. Maybe it accepts the hyphen format too? Not sure, but would check.


If it is evaluating to nil, then do we even need to set the locale? 

Also any reason not to use the currentLocal? Even if they are getting content in english, chrome is usually still in the users device language.